src/service/search/searchService.ts

```diff
@@ -0,0 +1,17 @@
+const searchRepo = require("./searchRepo");
+
+
+const businessFuzzySearch = async (businessName: string) => {
+    if (!businessName) throw Error("Business name is required");
+    try {
+      return await searchRepo.businessSearch(
```

Comment:
You sending an uppercased business name in the service and then uppercasing it again in the repo?